Lib/fontTools/misc/loggingTools.py

```diff
@@ -463,6 +463,13 @@ def emit(self, record):
 	def createLock(self):
 		self.lock = None
 
+	def match(self, regexp):
```

Comment:
I wanted to mimic the `match` method of pytest ExceptionInfo object:
https://docs.pytest.org/en/latest/builtin.html#_pytest._code.ExceptionInfo.match

I was thinking of returning `False` instead of `assert 0` and let the user choose to do `assert captor.match(...)` when writing pytest tests, or `self.assertTrue(captor.match(...))` when writing unittest tests. But then one would not get the error message, "Pattern '%s' not found...".

Ok, I'll go for `assertMatch`.